src/components/Room/Room.test.tsx

```diff
@@ -8,6 +8,9 @@ import { Room } from './'
 const mockUserId = 'user-id'
 const mockRoomId = 'room-123'
 
+window.AudioContext = jest.fn().mockImplementation(() => {
+  return {}
+})
```

Comment:
In order to test this functionality, we could simulate the receipt of a message and verify that `window.AudioContext` was called appropriately. However, the testing of this component is quite minimal so far. 😅 I don't think it's fair to ask you add tests for this given the current level of test coverage, so I think we can defer it for now unless you're feeling particularly ambitious.